code/modules/paperwork/paper_sticky.dm

```diff
@@ -78,6 +78,8 @@
 	icon = 'icons/obj/stickynotes.dmi'
 	color = COLOR_YELLOW
 	slot_flags = 0
+	layer = ABOVE_WINDOW_LAYER
+	pixel_z = 0
```

Comment:
Why are you zeroing this out here? Isn't it already 0?

Reply:
As I understand it, this is necessary so that the icons do not go beyond the boundaries of the screen. This fix not mine, I just ported it.

Reply:
This won't do anything, `pixel_z` is already 0 on this object definition. It might do something you you put it in logic or something, but not here.